libs/components/forms/testing/src/checkbox/checkbox-group-harness.ts

```diff
@@ -59,10 +64,15 @@ export class SkyCheckboxGroupHarness extends SkyComponentHarness {
   /**
    * Gets the help popover content.
    */
-  public async getHelpPopoverContent(): Promise<
-    TemplateRef<unknown> | string | undefined
-  > {
-    return await (await this.#getHelpInline()).getPopoverContent();
+  public async getHelpPopoverContent(): Promise<string | undefined> {
+    const content = await (await this.#getHelpInline()).getPopoverContent();
+
+    /* istanbul ignore if */
+    if (typeof content === 'object') {
+      throw Error('Unexpected template ref');
```

Comment:
Sandhya added `return content as string | undefined` in a previous PR; I think that's a tad better since this is exclusively a typings issue. In the case above, we're adding a needless `if` check to the resulting JavaScript bundle that's really only a "safety check" brought on by the types system.

Reply:
changed